File: vane-regions/src/main/java/org/oddlama/vane/regions/menu/RegionMenuTag.java
package org.oddlama.vane.regions.menu;

import java.util.UUID;

public class RegionMenuTag {

    private final UUID regionId;

    public RegionMenuTag(final UUID regionId) {
        this.regionId = regionId;
    }

    public UUID regionId() {
        return regionId;
    }
}

File: vane-regions/src/main/kotlin/org/oddlama/vane/regions/menu/RegionMenuTag.kt
package org.oddlama.vane.regions.menu

import java.util.*

/**
 * Lightweight menu tag carrying a selected region id.
 */
class RegionMenuTag(private val regionId: UUID?) {
    /**
     * Returns the associated region id.
     */
    fun regionId(): UUID? = regionId
}
